test/views/transit/AddAnotherOfficeOfTransitPageSpec.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.transit
+
+import pages.behaviours.PageBehaviours
+import pages.sections.transit.AddAnotherOfficeOfTransitPage
+
+class AddAnotherOfficeOfTransitPageSpec extends PageBehaviours {
+
+  "AddAnotherOfficeOfTransitYesNoPage" - {
```

Comment:
Very minor but this could be changed to match the correct name of the page object (`AddAnotherOfficeOfTransitPage`)